dockerfile.dev

```diff
@@ -13,6 +13,12 @@ FROM openjdk:8
 # Copy complete python container into java container
 COPY --from=python-container / /
 
+# Fetch the latest tagged licenses from Github for use in the Java license compares
+RUN apt-get update \
+    && apt-get -y install git
+RUN mkdir /licenses && cd /licenses && git clone https://github.com/spdx/license-list-data.git --depth=1 && cd license-list-data && git fetch --tags && git checkout $(git describe --tags `git rev-list --tags --max-count=1`)
+RUN mkdir /licenses/current && cp /licenses/license-list-data/website/*.jsonld /licenses/current && cp /licenses/license-list-data/website/licenses.json /licenses/current && cp /licenses/license-list-data/website/exceptions.json /licenses/current
+
```

Comment:
We can add ` rm -rf /var/lib/apt/lists/*` for cleaning up.

Reply:
Added the suggested command